### ISSUE TITLE
Method to preserve saved queries when cache is wiped

### ISSUE THREAD

**Issue by soileaud** [jetstreamapp/jetstream#1034]
**Describe the problem you are trying to solve**

When clearing browser cache (or getting a new machine) I have lost my saved queries

**Describe the solution you'd like**

If storage on the server is not an option, I would love the ability to manually export/import the queries to/from a file for backup. This could also allow users to share a library of queries with others.

**Comment by solo-1234** [comment]
I'd like it if the saved queries (and history?) would be stored in my user account instead of the browser cache, so that it would sync/persist across computers.

**Comment by josephdesch** [comment]
Agreed, this would be very helpful. There are regular queries the I run that I don't save because I know they'll get wiped out.

**Comment by Ckeds3** [comment]
This is crucial. I love everything about Jetstream except the risk of losing queries by clearing browser cache and when I get a new laptop every 2 years. I would even pay for it.

**Comment by paustint** [comment]
This feature is delivered and is part of the Pro plan
